projects/components/src/table/renderers/data-cell/table-data-cell-renderer.component.ts

```diff
@@ -29,7 +30,7 @@ import { TableCellRendererService } from '../table-cell-renderer.service';
         (popoverOpen)="this.popoverOpen = $event"
       ></htc-filter-button>
       <div class="cell-renderer-content" [ngClass]="this.alignment" (click)="this.onClick()">
-        <ng-container *ngComponentOutlet="this.rendererConstructor; injector: this.rendererInjector"></ng-container>
+        <div #cellRenderer></div>
```

Comment:
nit - I don't think this extra div is needed, if you move #cellRenderer to 32 it should be the equivalent dom to before

Reply:
Wasn't working before due to some css issues, but I'll see if I can adjust the styling and remove the div.